docker/prometheus/prometheus-server/Dockerfile.j2

```diff
@@ -4,7 +4,7 @@ LABEL maintainer="{{ maintainer }}" name="{{ image_name }}" build-date="{{ build
 {% block prometheus_server_header %}{% endblock %}
 
 {% block prometheus_server_repository_version %}
-ENV prometheus_version=1.7.1
+ENV prometheus_version=1.8.2
```

Comment:
ah, right, this is to avoid config changes I guess

Reply:
Yeah, could have done this via config changes, but it breaks the kolla-ansible side so thought it was going to be simpler to bake it in.